addons/mail/static/src/js/text_emojis_field.js

```diff
@@ -33,18 +31,11 @@ var FieldEmojiCommon = {
      * @override
      */
     on_attach_callback: function () {
-        var self = this;
-
         if (!this.$emojisIcon) {
-            this.$emojisIcon = $(QWeb.render(
-                'mail.EmojisDropdown', {
-                    widget: this,
-                    emojisDropdownId: 'emojis_dropdown'
-                }
-            ));
-            this.$emojisIcon.find('.o_mail_emoji').on('click', function (ev) {
-                self._onEmojiClick(ev);
-                self._isDirty = true;
+            this.$emojisIcon = $(QWeb.render('mail.EmojisDropdown', {widget: this}));
+            this.$emojisIcon.find('.o_mail_emoji').on('click', (ev) => {
```

Comment:
Please no ES6 yet, we will wait for OWL to keep some consistency in the code for now.